src/config/global-config.c

```diff
@@ -56,6 +58,29 @@ int Read_GlobalSK(XML_NODE node, void *configp, __attribute__((unused)) void *ma
                 merror(XML_VALUEERR, node[i]->element, node[i]->content);
                 return (OS_INVALID);
             }
+            if (strcmp(node[i]->attributes[0], xml_ignore_frecuency) == 0) {
```

Comment:
Please use:
```c
int j;
for (j = 0; node[i]->attributes && node[i]->attributes[j]; ++j) {
    if (strcmp(node[i]->attributes[j], xml_ignore_frecuency) == 0) {
    // ... use node[i]->values[j]
    } else if (strcmp(node[i]->attributes[j], xml_ignore_time) == 0) {
    // ... use node[i]->values[j]
    } else {
        merror(XML_INVATTR, node[i]->attributes[j], node[i]->element);
        return OS_INVALID;
    }
}
```

This will allow any order in the attributes and optional attributes. It will also prevent a segmentation fault if no such attributes are present.